examples/connectivity/plot_connectivity_measures.py

```diff
@@ -0,0 +1,95 @@
+"""
+Comparing different functional connectivity measures
+====================================================
+
+This example compares different measures of functional connectivity between
+regions of interest : correlation, partial correlation, as well as a robust
+dispersion measure. The resulting connectivity coefficients are used to
+classify ADHD vs control subjects and the robust dispersion outperforms the
+standard measures.
+
+"""
+
+# Fetch dataset
+import nilearn.datasets
+atlas = nilearn.datasets.fetch_atlas_msdl()
+dataset = nilearn.datasets.fetch_adhd()
+
+# Extract regions time series signals
+import nilearn.input_data
+masker = nilearn.input_data.NiftiMapsMasker(
+    atlas.maps, resampling_target="maps", detrend=True,
+    low_pass=None, high_pass=None, t_r=2.5, standardize=False,
+    memory='nilearn_cache', memory_level=1)
+subjects = []
+sites = []
+adhds = []
+for func_file, phenotypic in zip(dataset.func, dataset.phenotypic):
+    # keep only 3 sites, to save computation time
+    if phenotypic['site'] in ['"NYU"', '"OHSU"', '"NeuroImage"']:
```

Comment:
@salma1601 you can prefix the strings with b, i.e. b'"NYU"', etc... Not great but this would make this example work with Python 3.

I still think the downloader should be fixed to not have the site names contain a quote.